README.nix.md

```diff
@@ -0,0 +1,92 @@
+# Using Nix with Forest
+
+This guide will help you get started with building and installing Forest using
+Nix.
+
+## Installing Nix
+
+The recommended way to install Nix is using Determinate Systems' Nix installer:
+
+1. Run the following command in your terminal:
+   ```bash
+   curl -fsSL https://install.determinate.systems/nix | sh -s -- install --determinate
+   ```
+
+2. After installation, restart your shell or run:
+   ```bash
+   . /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh
+   ```
+
+See [Determinate Systems' Nix installation
+guide](https://docs.determinate.systems/getting-started/individuals) for more
+information.
+
+## Building Forest with Flakes
+
+Forest uses Nix flakes for reproducible builds. To build Forest:
+
+1. Clone the Forest repository
+2. Run the following command in the repository root:
+   ```bash
+   nix build
```

Comment:
I got the below errors running `nix build` locally. Do I need to sign in or set a token?

```
warning: unable to download 'https://cache.flakehub.com/nix-cache-info': HTTP error 401

         response body:

         {"code":401,"error":"Unauthorized","message":"Unauthorized."}
```

Reply:
It's just a warning. To use the cache, you have to log in to FlakeHub.

Reply:
Thanks, maybe it's worth mentioning in the doc as well to avoid surprises.